### DISCUSSION TITLE
how to set GRAY in config of otb100 when training、

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#192] [General]
how to set GRAY in config of otb100 when training、

**Comment by ZhiyuanChen** [comment]
Sorry but I am having troubles to understand your question.
You may use Chinese or other languages if you find it's easier.

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
